docs/resources/organization_connection.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
This was generated with the `make docs-gen RESOURCE=organization_connection` command.